Recruit-Training/Embedded-Recruit-Training/Core/Src/GreenLedToggleTask.c

```diff
@@ -3,15 +3,35 @@
 void greenLedToggleTask(void const* arg)
 {
     //One time osDelayUntil initialization
+    static const uint32_t GREEN_LED_STATUS_STDID = 0xDDD;
+    static const uint32_t GREEN_LED_TOGGLE_FREQ = 100;
     uint32_t prevWakeTime = osKernelSysTick();
 
     osMutexId_t* canMutex = (osMutexId_t*)arg;
 
     for (;;)
     {
         //TODO: Add GREEN_LED_TOGGLE_FREQ to prevWakeTime
+        prevWakeTime += GREEN_LED_TOGGLE_FREQ;
         osDelayUntil(prevWakeTime);
         //TODO: Check blue toggel flag and toggle blue LED
+        if (greenFlag == 1){
```

Comment:
You could also just `if(greenFlag)` because in C as long as the value is greater than 0 it would evaluate to true.